lib/lightning_web/live/workflow_live/edit.ex

```diff
@@ -172,6 +176,7 @@ defmodule LightningWeb.WorkflowLive.Edit do
 
       <div class="relative h-full flex" id={"workflow-edit-#{@workflow.id}"}>
         <div class="flex-none" id="job-editor-pane">
+          <.WorkflowStoreSync />
```

Comment:
Yes!!!

Reply:
Yup. works beautiful now. 
1. Gets store updates from phoenix downwards
2. Both WorkflowEditor & JobEditor share the same store regardless of which was mounted first
3. I'm currently not pushing updates back to phoenix yet. **in progress**...